purchases/src/main/kotlin/com/revenuecat/purchases/google/usecase/BillingClientUseCase.kt

```diff
@@ -125,19 +126,19 @@ internal abstract class BillingClientUseCase<T>(
         }
     }
 
-    private fun backoffOrErrorIfUseInSession(
+    private fun backoffOrErrorIfManyRetries(
         onError: (BillingResult) -> Unit,
         billingResult: BillingResult,
     ) {
-        if (useCaseParams.appInBackground) {
-            log(LogIntent.GOOGLE_WARNING, BillingStrings.BILLING_SERVICE_UNAVAILABLE_BACKGROUND)
-            if (retryBackoffMilliseconds < RETRY_TIMER_MAX_TIME_MILLISECONDS) {
-                retryWithBackoff()
-            } else {
-                onError(billingResult)
-            }
+        log(LogIntent.GOOGLE_WARNING, BillingStrings.BILLING_SERVICE_UNAVAILABLE.format(useCaseParams.appInBackground))
```

Comment:
nitpick I know this was here before, but I think this log should go in line 76. This function could be call in cases different to `billing unavailable`

Reply:
Hmm I think this function is only called when we receive `ServiceUnavailable` from google though? (I might be missing something)

Reply:
Yes, it's only called from there, but this log is the only thing that makes it not reusable from another case, so what I meant is that if someone calls the function _in the future_ and doesn't look at the implementation, this log will be wrong.

You can leave it as is. I was triggered by the fact that the name of the function is `backoffOrErrorIfManyRetries` and not `backoffOrErrorIfManyRetriesWhenBillingUnavailable`

Reply:
Oh right, I just renamed to try to make it clearer. Thanks!